OpenQA/Isotovideo/Utils.pm

```diff
@@ -92,7 +94,8 @@ sub checkout_git_repo_and_branch ($dir_variable, %args) {
     else {
         bmwqemu::diag "Skipping to clone '$clone_url'; $local_path already exists";
     }
-    return $bmwqemu::vars{$dir_variable} = path($local_path)->to_abs->to_string;
+    return $bmwqemu::vars{$dir_variable} = path($local_path)->to_abs->to_string unless $args{repo};
+    return undef;
```

Comment:
I suggest to make it:

```suggestion
    return $args{repo} ? undef : $bmwqemu::vars{$dir_variable} = path($local_path)->to_abs->to_string;
```

This way we use the positive if-condition and make it obvious that we always want to return something valid but distinguish based on an option.